src/main/kotlin/test/ktortemplate/core/persistance/sql/CarRepositoryImpl.kt

```diff
@@ -4,62 +4,47 @@ import org.jetbrains.exposed.sql.ResultRow
 import org.jetbrains.exposed.sql.deleteWhere
 import org.jetbrains.exposed.sql.insert
 import org.jetbrains.exposed.sql.select
-import org.koin.core.KoinComponent
-import org.koin.core.inject
-import test.ktortemplate.conf.database.DatabaseConnection
 import test.ktortemplate.core.model.Car
 import test.ktortemplate.core.model.CarSaveCommand
 import test.ktortemplate.core.persistance.CarRepository
 import test.ktortemplate.core.utils.pagination.PageRequest
 import test.ktortemplate.core.utils.pagination.createSorts
 import test.ktortemplate.core.utils.pagination.fromFilters
 
-class CarRepositoryImpl : CarRepository, KoinComponent {
+class CarRepositoryImpl : CarRepository {
```

Comment:
Note that in the repositories we're no longer creating a new transaction, the callers of the repositories are responsible for ensuring the desired transactional semantics. This means that if you forget to open a transaction this code will fail - on the other hand you are now forced to think about the kind of transactional semantics you actually want, instead of getting something that works... until it doesn't.